exercises/clock/src/Clock.hs

```diff
@@ -1,4 +1,4 @@
-module Clock (clockHour, clockMin, fromHourMin, toString) where
+module Clock (clockHour, clockMin, fromHourMin, toString, addDelta) where
```

Comment:
for alphabetical order, I think addDelta should go first. Unless there's no actual reason to use a particular ordering